cypress/integration/rendering/gantt.spec.js

```diff
@@ -14,18 +14,18 @@ describe('Gantt diagram', () => {
       excludes weekdays 2014-01-10
 
       section A section
-      Completed task            :done,    des1, 2014-01-06,2014-01-08
-      Active task               :active,  des2, 2014-01-09, 3d
-      Future task               :         des3, after des2, 5d
-      Future task2               :         des4, after des3, 5d
+      Completed task                      :done, des1, 2014-01-06,2014-01-08
+      Active task                         :active, des2, 2014-01-09, 3d
+      Future task                         :des3, after des2, 5d
+      Future task2                        :des4, after des3, 5d
```

Comment:
These formatting changes should be reverted. These could have been tests cases to check inconsistent spacing. 

So please revert changes to existing test, and add new standalone cases for until.